dask/base.py

```diff
@@ -139,11 +141,20 @@ def normalize_function(func):
 
 
 normalize_token = Dispatch()
-normalize_token.register((int, float, str, tuple, list), lambda a: a)
-normalize_token.register(object,
-        lambda a: normalize_function(a) if callable(a) else a)
+normalize_token.register((int, float, str, unicode, bytes, tuple, list,
```

Comment:
Slightly more reasonable suggestion:

`tokenize` takes `*args`, and `**kwargs`. I suggest recursing one level into each argument, as well as each value in `kwargs`. Not as bad combinatorially, but should be good enough for most cases.